### PR TITLE
Fixed the logout button at the bottom of the admin dashboard and fixe…

### PR DESCRIPTION
**Pull request by grahamD-cyber**
Fixed the logout button at the bottom of the admin dashboard and fixed the background issues we were having all term.

### CODE REVIEW

**Comment by grahamD-cyber** [comment]
All existing Admin issues have been resolved by this push. I fixed the responsiveness of all components on the admin side.

File: public/forgotPass.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <title>Forget Password Page</title>
     <link rel="stylesheet" href="./css/styles.css" />
     <link rel="stylesheet" href="./css/forgotPass.css">
-    <link rel="stylesheet" href="./css/admin.css">
+    <link rel="stylesheet" href="./css/account.css">
     <script src="/socket.io/socket.io.js"></script>
     <script src="js/chat.js"></script>
```
Comment (kelsey-debug) [inline public/forgotPass.html]: the socket io and chat.js script need to be removed

**Review by kelsey-debug — CHANGES_REQUESTED** [review]
Looks good except remove the script tags for socketio and chat.js implementation. That could cause an issue.

File: public/forgotPass.html (hunk 1 above)
Comment (grahamD-cyber) [reply public/forgotPass.html]: Got it. I was wondering about that. Is that just in the forgotPass.html file?

**Review by kelsey-debug — APPROVED** [review]

**Pull request merged**